docs/CHANGELOG.asciidoc

```diff
@@ -181,6 +181,18 @@ You must do a force unenroll to remove the {agent} and invalidate the API keys,
 or unenrollment hangs indefinitely. https://github.com/elastic/fleet-server/issues/380[#380]
 ====
 
+.*TSVB* references to the index pattern are not embedded when exporting a saved object
+[%collapsible]
+====
+*Details*  +
+This issue was reported at https://github.com/elastic/kibana/issues/103059[#103059] and was originally 
```

Comment:
*TSVB* references to the index pattern are not embedded when exporting a saved object.